### PR TITLE
Feat/2645 prepare grid multiple w

### PR DESCRIPTION
**Pull request by adipaul1981**

### CODE REVIEW

File: src/components/UserDashboard/SavedQueries/index.js [hunk 1]
```diff
@@ -169,70 +166,83 @@ class SavedQueries extends React.Component {
...
+    this.setState({ activeTab: key });
+  };
+
   render() {
     const {
       state: { queries: fileQueries, loadingQueries },
       virtualStudies,
     } = this.props;
 
+    const tabList = [
```
Comment (evans-g-crsj) [inline src/components/UserDashboard/SavedQueries/index.js:198]: can this be extracted in a class method? such as generateTabList

File: src/components/UserDashboard/SavedQueries/index.js [hunk 2]
```diff
@@ -169,70 +166,83 @@ class SavedQueries extends React.Component {
...
+
+    const contentList = {
+      participantTab: this.renderParticipantQueries(),
+      fileTab: this.renderFileQueries(),
+    };
+
     return (
-      // TODO EXTRACT DashboardCard to UserDashboard/index.js
-      <>
+      <div className={'saved-queries-container'} style={{ display: 'contents' }}>
```
Comment (evans-g-crsj) [inline src/components/UserDashboard/SavedQueries/index.js:227]: in Css?

File: src/components/UserDashboard/CavaticaProjects/index.js [hunk 3]
```diff
@@ -31,54 +41,49 @@ const generateTabsContent = ({
...
+  const ConnectedComp = (cardProps) => (
+    <Connected tabToCreate={tabToCreate(cardProps)} projects={projects} loading={loading} />
+  );
+  const CreateComp = (cardProps) => (
+    <Create
+      onProjectCreated={onProjectCreated({ cardProps, refresh })}
+      onProjectCreationCancelled={onProjectCreationCanceled(cardProps)}
+    />
+  );
+  const ResultComp = () => () => (
```
Comment (evans-g-crsj) [inline src/components/UserDashboard/CavaticaProjects/index.js:56]: I think that using closure here is a smell...it should be avoided if possible

**Review by evans-g-crsj — CHANGES_REQUESTED** [review]

File: src/components/UserDashboard/SavedQueries/SavedQueries.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+@import '../../../style/themes/default/colors';
```
[the remark this answers is not in the document]
Comment (adipaul1981) [reply src/components/UserDashboard/SavedQueries/SavedQueries.scss:1]: yes

File: src/components/UserDashboard/SavedQueries/index.js (hunk 1 above)
Comment (adipaul1981) [reply src/components/UserDashboard/SavedQueries/index.js:198]: im not sure a method is required... a list is just fine. There is only two possible tabs anyway.

**Review by evans-g-crsj — APPROVED** [review]

**Pull request merged**